### ISSUE TITLE
mAP drops when changing backbone to ViT-L-14

### ISSUE THREAD

**Issue by DFLAG-NEU** [Syliz517/CLIP-ReID#56]
Thank you for your outstanding work and for sharing such a great codebase.  While experimenting with your method, I replaced the original backbone with ViT-L-14.  The pre-trained model that I downloaded from https://openaipublic.azureedge.net/clip/models/b8cca3fd41ae0c99ba7e8951adf17d267cdb84cd88be6f7c2e0eca1737a03836/ViT-L-14.pt. 

However, I observed a significant drop in mAP performance during the evaluation. I suspect this may be due to differences in the backbone architecture requiring adjustments in training settings. Could you please advise on what modifications I should consider when re-training the model? For example, should I adjust the learning rate, optimizer, number of epochs, or any other settings to better suit ViT-L-14?

Thank you very much for your time and assistance.

**Comment by edwardnguyen1705** [comment]
Hi @DFLAG-NEU ,
I encounter the same issue. Have you found out something?

**Comment by edwardnguyen1705** [comment]
Dear @Syliz517 ,
Thanks for sharing your great work.
Regarding the above issue, do you have any suggestion?
Thanks for your time.

**Comment by shawyang3753** [comment]
不管是vit-l，其他backbone也准确率很低
